src/components/HighlightableMenuItem.tsx

```diff
@@ -0,0 +1,36 @@
+import type {ForwardedRef} from 'react';
+import {forwardRef} from 'react';
+import type {View} from 'react-native';
+import {StyleSheet} from 'react-native';
+import useAnimatedHighlightStyle from '@hooks/useAnimatedHighlightStyle';
+import useThemeStyles from '@hooks/useThemeStyles';
+import MenuItem from './MenuItem';
+import type {MenuItemProps} from './MenuItem';
+
+type Props = MenuItemProps & {
+    highlighted?: boolean;
```

Comment:
Please add comment here for compliance sake i.e. and each prop has a /** comment above it */

Reply:
Handled.